README.md

```diff
@@ -41,7 +41,7 @@ Finally, this implementation uses the [ASP.NET Core Docker sample web app](https
 * [Azure KeyVault Secret Store CSI Provider](https://github.com/Azure/secrets-store-csi-driver-provider-azure)
 * [Kured](https://docs.microsoft.com/azure/aks/node-updates-kured)
 
-![TODO: apply meaningful assistive technology description](https://docs.microsoft.com/azure/architecture/reference-architectures/containers/aks/secure-baseline/images/baseline-network-topology.png)
+![Network diagram depicting a hub-spoke network with two peered VNets, each with three labeled subnets.](https://docs.microsoft.com/azure/architecture/reference-architectures/containers/aks/_images/baseline-network-topology.png)
```

Comment:
```suggestion
![Network diagram depicting a hub-spoke network with two peered VNets, each with three labeled subnets and main Azure resources.](https://docs.microsoft.com/azure/architecture/reference-architectures/containers/aks/_images/secure-baseline-architecture.png)
```

I would suggest going with https://github.com/MicrosoftDocs/architecture-center-pr/blob/7cf4633c790e0b8257edd657e1b39a95ce592360/docs/reference-architectures/containers/aks/_images/secure-baseline-architecture.png as this is a more comprehensive one

Reply:
Funny you recommend that, because when I wrote the first alt text pass I was describing this exact image, and then I decided to "do a quick check" to make sure it actually was this image ... and it wasn't, lol.  So I made a couple quick modifications.  I agree this would be a better choice for this purpose. :)